django/db/backends/oracle/creation.py

```diff
@@ -12,9 +13,26 @@
 
 class DatabaseCreation(BaseDatabaseCreation):
 
+    @cached_property
+    def _maindb_connection(self):
+        """
+        This is analogous to other backends' `_nodb_connection` property,
+        which allows access to an "administrative" connection which can
+        be used to manage the test databases.
+        For Oracle, the only connection that can be used for that purpose
+        is the main (non-test) connection.
+        """
+        settings_dict = settings.DATABASES[self.connection.alias]
+        user = settings_dict.get('SAVED_USER') or settings_dict['USER']
```

Comment:
On Oracle, to create a test database, you need to create a new user. Then, in order to use it, you need to change the settings to refer to your new user; but you need to revert to the original user in order to do the clean-up (remove the test user). So the original user/password is saved in SAVED_USER and SAVED_PASSWORD.

I will try to find a good place to clarify this.